prime-dotnet-webapi/Controllers/HealthAuthoritiesController.cs

```diff
@@ -86,18 +86,18 @@ public async Task<ActionResult> GetAuthorizedUsers(int healthAuthorityId)
             return Ok(users);
         }
 
-        // GET: api/health-authorities/sites
+        // GET: api/health-authorities/site-lists
         /// <summary>
-        /// Gets all sites for any health authority.
+        /// Gets all site-lists for health authority.
         /// </summary>
-        [HttpGet("sites", Name = nameof(GetAllHealthAuthoritySites))]
+        [HttpGet("site-lists", Name = nameof(GetAllHealthAuthoritySiteLists))]
         [Authorize(Roles = Roles.ViewSite)]
         [ProducesResponseType(StatusCodes.Status401Unauthorized)]
         [ProducesResponseType(StatusCodes.Status403Forbidden)]
-        [ProducesResponseType(typeof(ApiResultResponse<IEnumerable<HealthAuthoritySiteViewModel>>), StatusCodes.Status200OK)]
-        public async Task<ActionResult> GetAllHealthAuthoritySites()
+        [ProducesResponseType(typeof(ApiResultResponse<IEnumerable<HealthAuthoritySiteListViewModel>>), StatusCodes.Status200OK)]
+        public async Task<ActionResult> GetAllHealthAuthoritySiteLists()
         {
-            return Ok(await _healthAuthoritySiteService.GetSitesAsync());
+            return Ok(await _healthAuthoritySiteService.GetSiteListsAsync());
```

Comment:
`.GetSitesAsync()` follows our current service naming convention, I wouldn't expect this has to change.